server/requirements.txt

```diff
@@ -0,0 +1 @@
+git+git://github.com/superdesk/newsroom-core.git@core#egg=Newsroom-Core
```

Comment:
the branch should be develop once https://github.com/superdesk/newsroom-core/pull/1 is merged right?

Reply:
yes, we can create develop branch after main things are merged and use develop in `requirements.txt`